types/react-star-ratings/README.md

```diff
@@ -0,0 +1,5 @@
+# Installation
```

Comment:
The `@types` publisher generates a README. You do not need to write one yourself.